app/services/plans/create_service.rb

```diff
@@ -77,6 +85,17 @@ def create_commitment(plan, args, commitment_type)
       )
     end
 
+    def create_usage_threshold(plan, args)
+      usage_threshold = plan.usage_thresholds.new(
+        threshold_display_name: args[:threshold_display_name],
+        amount_cents: args[:amount_cents],
+        amount_currency: args[:amount_currency],
```

Comment:
Do we allow specifying a different currency from the plan's currency?

Reply:
Yes we do, it was discussed [here](https://www.notion.so/Dive-in-Progressive-billing-tresholds-d9b176432ea8406ebc8a6af8103e358c?d=0acb194edf0b4a0b99f4072045c73ddd&pvs=4#47f09c60261445b9ad84c8d7fc5432bb).

Reply:
Okay 👍 I've moved the discussion to the dive-in, allowing thresholds with different currencies would complicate the calculation. We should discuss with product I think on how to proceed :)

Reply:
We'll always use plan's currency for tresholds.